extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -57,6 +49,25 @@ export class AmpVizVega extends AMP.BaseElement {
     /** @private {?string} */
     this.src_ = this.element.getAttribute('src');
 
+    /** @private {boolean} */
+    this.useDataWidth_ = this.element.hasAttribute('use-data-width');
```

Comment:
Not directly but these are tied to the `use-data-width` and `use-data-height` attributes. For now, they are immutable after buildCallback